tests/files/tests.py

```diff
@@ -200,6 +200,24 @@ def test_shared_lock(self):
             self.assertIs(locks.unlock(f1), True)
             self.assertIs(locks.unlock(f2), True)
 
+    def test_open_supports_full_signature(self):
+        """
+        Django File.open should support full signature of Python builtin open
+        #34642 File.open to support different encodings
```

Comment:
This docstring sounds unnecessary, we should at least simplify it, e.g.
```suggestion
        File.open supports full signature of Python's open.
```